_includes/configuration/body-class.md

```diff
@@ -1,6 +1,6 @@
 ## body_class
 
-This option enables you to specify a class for the body of each editor instance. This class can then be used to do {{site.productname}} specific overrides in your `content_css`. There is also a specific `mceForceColors` class that can be used to override the text and background colors to be black and white.
+Use the `body_class` option to add a class to the body of each editor instance. This class can be used to override the styles added by `content_css`. The `body_class` will be removed if the editor is removed and will not be submitted with the content on a `POST` request.
```

Comment:
The post request part is confusing here, as TinyMCE doesn't POST any content itself. I'd just say when getting the content from the editor, or something similar to that.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1643/commits/e953adb2029d4b6efc8b5c8c4edc17b617e4c235